# ########## ymglp params ##########

YmgGPHybROS:
  path_resolution: 10.0
  stuck_timeout: 10.0
  navfn_goal_dist: 5.0
  recovery_dist: 2.0

